run.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+stack exec hylide examples/Example.hs
```

Comment:
Is this file necessary? It's not referenced in the README. 

Omit this file and you're golden :)